conf/application.conf

```diff
@@ -15,7 +15,7 @@
 include "common.conf"
 
 application.global = uk.gov.hmrc.tai.config.ApplicationGlobal
-application.router=app.Routes
+play.http.router=app.Routes
```

Comment:
What this change about?

Reply:
application.router is deprecated. We were getting warning messages when running tests advising to use play.http.router instead. It seems to do exactly the same thing, so I spoke to @RaminEsfandiari and he said it should be ok to change.

Reply:
There is an argument that things like this and the reformatting of the `conf/tai.routes` should be done in separate PRs so that it doesn't add noise to the work you are actually doing. Maybe thats being abit pedantic though when there isn't many changes